### ISSUE TITLE
Use of DataLoader to solve exorbitant query times

### ISSUE THREAD

**Issue by illeatmyhat** [graphql-python/graphene-mongo#45]
There's a dramatic reduction in query performance when using GraphQL with graphene-mongo, as opposed to a regular REST API doing something like `list(collection.objects.all())` and sending the entire document set. From 300ms to over 6 seconds to retrieve one field from 100 elements.

I believe that using the DataLoader pattern is the expected solution to the N+1 problem, but actually attempting to use it will cause a runtime error
```
graphql.error.located_error.GraphQLLocatedError: Data loader batch_load_fn function raised an Exception: AttributeError("type object 'Person' has no attribute 'objects'",)
```

**Comment by abawchen** [comment]
@illeatmyhat 👍 
I have time to take care it now :bowtie: 
Would you please provide your snippet code? Thanks.

**Comment by illeatmyhat** [comment]
Unfortunately I've put GraphQL on the backburner since this issue and don't have the code.

**Comment by abawchen** [comment]
@illeatmyhat : It supports relay query with DataLoader by #91, but not sure solves your error or not, and I think it needs to do some change of the backend implementation you work with.

Since you don't have the code that I can't investigate further 😟

**Comment by abawchen** [comment]
@illeatmyhat : Close this one since it's stale, and maybe it has been addressed in #115 and #125. I will try to follow up these 2 items.
